projects/engine-node/tests/child-process.ts

```diff
@@ -81,7 +80,7 @@ describe('ChildProcess plugin', () => {
     try {
       await plugin.activate();
       (plugin.process as any).connected = false
-      plugin['postMessage']({ name: 'child-process' })
+      plugin['send']({ name: 'child-process' })
```

Comment:
what's the reasoning behind this renaming?

Reply:
It was either `postMessage`, the iframe style, or `send`, the websocket style. I choose the shortest.